### PR TITLE
[Declarative Link Capturing article] Deprecate article

### PR DESCRIPTION
**Pull request by tomayac**
Changes proposed in this pull request:

- Deprecate article.

### CODE REVIEW

File: src/site/content/en/blog/declarative-link-capturing/index.md [hunk 1]
```diff
@@ -26,9 +26,16 @@ origin_trial:
...
+{% Aside 'warning' %} Declarative Link Capturing was part of the
+[capabilities project](https://web.dev/fugu-status/)
+and is no longer pursued. The feature will be reincarnated as `launch_handler` and `scope_extensions`.
+Watch this site for updates. {% endAside %}
+
+{% Details %}
+
+{% DetailsSummary %}
+Original article
+{% endDetailsSummary %}
```
Comment (jpmedley) [inline src/site/content/en/blog/declarative-link-capturing/index.md:38]: What is this?

**Review by jpmedley — CHANGES_REQUESTED** [review]

File: src/site/content/en/blog/declarative-link-capturing/index.md (hunk 1 above)
Comment (tomayac) [reply src/site/content/en/blog/declarative-link-capturing/index.md:38]: Just added you on the email thread.

Comment (jpmedley) [reply src/site/content/en/blog/declarative-link-capturing/index.md:38]: I saw that thanks. I had to read that email thread to understand the context of this. Web.dev readers won't have that context. In any case, shouldn't this say something like, "The features described in this article are no longer being pursued"?

Comment (tomayac) [reply src/site/content/en/blog/declarative-link-capturing/index.md:38]: Please see the [staged preview](https://deploy-preview-5950--web-dev-staging.netlify.app/declarative-link-capturing/), it does exactly that on the top and in the title and leaves the original article accessible. (It’s the same for both articles.)

Comment (jpmedley) [reply src/site/content/en/blog/declarative-link-capturing/index.md:38]: So it makes the article content collapsable. To what end?

Comment (tomayac) [reply src/site/content/en/blog/declarative-link-capturing/index.md:38]: Just the entire article, apart from the warning aside.

**Pull request closed without merging**